apps/prairielearn/elements/pl-big-o-input/pl-big-o-input.py

```
@@ -73,12 +74,20 @@ def prepare(element_html: str, data: pl.QuestionData) -> None:
     pl.check_answers_names(data, name)
 
-    variables = psu.get_items_list(
+    single_variable = psu.get_items_list(
         pl.get_string_attrib(element, "variable", VARIABLES_DEFAULT)
     )
+    variables = psu.get_items_list(
+        pl.get_string_attrib(element, "variables", VARIABLES_DEFAULT)
+    )
+
+    if len(single_variable[0]) != 0 and len(variables[0]) != 0:
+        raise ValueError("variable is deprecated, use only 'variables' property")
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)